INSTRUCTIONS.md

```diff
@@ -15,6 +15,21 @@
 
    To learn more about solana client binary usage [click here](https://github.com/Chainflow/solana-mission-control/blob/main/docs/prereq-manual.md#install-solana-client).
 
+   - Instructions to store solana client binary path in solana.env file, This environment file will be used to add solana client binary in solana_mc.service file.
```

Comment:
User willl export the solana binary path.
We have to read the value and copy it in a file.

We should not ask for them to copy the value to .env file.

In manusal instructions,
- add commands to cp the the path to .env file.
In script
- read the exported value from user and store it in .env file.

Reply:
Sure @PrathyushaLakkireddy  I've updated Instructions.